package-lock.json

```diff
@@ -58,6 +58,41 @@
         "import-cwd": "2.1.0",
         "mocha": "5.2.0",
         "nyc": "13.1.0"
+      },
```

Comment:
There's something wrong with the lock file, lots of dependencies are being downgraded/modified

Reply:
Will look into it 👍
I cerated the PR mainly to get the netlify build to validate the set env in build step. Noticed that you cant just stringfy an error so some more work to be done for this PR

Reply:
added a CookieConsent with similar styling as core.qlik.com